DecipherNow/gm-fabric-dashboard
# 1330 Render package.json version in app

Resolves #1330 

## Proposed Changes

  - Have the app treat the version in the package.json as the definitive one and show that in the app
  - Add new tooling around dotenv to expand npm variables available during build time

## How to Test

  - Change the version in package.json
  - `npm start`
  - Confirm that the version is what you expect
